src/components/inputBox.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { mediaQuery, calcSize } from 'styles/media';
+import color from 'styles/colors';
+
+interface InputBoxProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  label: string | undefined;
+  subinfo: string | undefined;
```

Comment:
```suggestion
  label?: string;
  subinfo?: string;
```

요렇게하면 동일한 표현이 가능할 것 같습니다!